extensions/blocks/donations/donations.php

```diff
@@ -40,29 +94,170 @@ function register_block() {
  * @return string
  */
 function render_block( $attr, $content ) {
+	// Keep content as-is if rendered in other contexts than frontend (i.e. feed, emails, API, etc.).
+	if ( ! jetpack_is_frontend() ) {
+		return $content;
+	}
+
 	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME, array( 'thickbox' ) );
 
-	require_once JETPACK__PLUGIN_DIR . '/modules/memberships/class-jetpack-memberships.php';
+	require_once JETPACK__PLUGIN_DIR . 'modules/memberships/class-jetpack-memberships.php';
+	require_once JETPACK__PLUGIN_DIR . 'class-jetpack-currencies.php';
+
 	add_thickbox();
 
 	$donations = array(
-		'one-time' => $attr['oneTimeDonation'],
-		'monthly'  => $attr['monthlyDonation'],
-		'annual'   => $attr['annualDonation'],
+		'one-time' => array_merge(
+			array(
+				'title' => __( 'One-Time', 'jetpack' ),
+				'class' => 'donations__one-time-item',
+			),
+			$attr['oneTimeDonation']
+		),
 	);
+	if ( $attr['monthlyDonation']['show'] ) {
+		$donations['1 month'] = array_merge(
+			array(
+				'title' => __( 'Monthly', 'jetpack' ),
+				'class' => 'donations__monthly-item',
+			),
+			$attr['monthlyDonation']
+		);
+	}
+	if ( $attr['annualDonation']['show'] ) {
+		$donations['1 year'] = array_merge(
+			array(
+				'title' => __( 'Yearly', 'jetpack' ),
+				'class' => 'donations__annual-item',
+			),
+			$attr['annualDonation']
+		);
+	}
+
+	$currency = $attr['currency'];
+
+	$classes = 'wp-block-jetpack-donations';
+	if ( ! empty( $attr['className'] ) ) {
+		$classes .= ' ' . $attr['className'];
+	}
```

Comment:
What do you think about relying on `Blocks::classes()` here?

Reply:
Another TIL! Much simpler now. See 7f767e5c2f2638d50d367e456f97317edf5b44e0